common/djangoapps/util/tests/test_keyword_sub_utils.py

```diff
@@ -123,4 +123,4 @@ def test_subbing_no_userid_or_courseid(self):
             (key, value) for key, value in self.context.iteritems() if key != 'user_id'
         )
         result = Ks.substitute_keywords_with_data(test_string, no_user_id_context)
-        self.assertEqual(test_string, result)
+        self.assertEqual(test_string, result)
```

Comment:
@shadinaif Please add the newline back. See the :no_entry_sign:? It shouldn't be there.

<kbd>![image](https://user-images.githubusercontent.com/645156/44955269-366b8900-aeb9-11e8-8940-a5682e60f956.png)</kbd>